watcher/watcher.go

```diff
@@ -60,13 +60,14 @@ type (
 	// from the watcher to the client.
 	//
 	// This is initialized when client starts watching for a given channel.
-	// Client can receive the event via the channel returned by Next method.
+	// Client can receive the event via the channel returned by EventStream
+	// method.
 	//
 	// This channel will be closed when client requests the watcher to stop
 	// watching or when there is an error. After the channel is closed, error
 	// message can be read using the Err method.
 	AdjudicatorSub interface {
-		Next() <-chan channel.AdjudicatorEvent
+		EventStream() <-chan channel.AdjudicatorEvent
```

Comment:
Updated documentation that `Multiple calls to the EventStream method returns the same channel instance.` in PR #212.

Reply:
I think multiplexing is not necessary in our use case, because the only consumer of the events is the watch function.

If at a later point in time, we have the need for consuming the events in multiple places, then we could implement multiplexing.